### ISSUE TITLE
Search in files with ctp extension

### ISSUE THREAD

**Issue by garfildalex6** [quasilyte/phpgrep#82]
Thanks for the work you've done.
When I try to search a file with the ctp extension, it doesn't find anything. As soon as I change the file extension to php, the search occurs.
The fact is that I have too many such files (this is a requirement of the CakePHP framework) and it is not possible to change the extensions for all files.
Please add the ability to search in such files.

**Comment by garfildalex6** [comment]
Tell me how to download the compiled file for Windows to check?

**Comment by quasilyte** [comment]
I released a v1.0.1 version with binaries.

If you want to build it from source, you would need Go toolchain on your machine and do:
```
git clone https://github.com/quasilyte/phpgrep.git
cd phpgrep
go build -o phpgrep.exe ./cmd/phpgrep
```
Or use a go install command:
```
go install github.com/quasilyte/phpgrep/cmd/phpgrep
```

**Comment by garfildalex6** [comment]
This is wonderful! Thanks a lot! Everything works as expected!
